### PR TITLE
Add timestamp to console output

### PR DESCRIPTION
**Pull request by externl**
This PR just adds a timestamp to the console output. It's otherwise impossible to tell when the logs are from.

### CODE REVIEW

File: src/Hello/Hello.cs [hunk 1]
```diff
@@ -34,7 +34,10 @@
 // Create a simple console logger factory and configure the log level for category IceRpc.
 using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
     builder
-    .AddSimpleConsole()
+    .AddSimpleConsole(options =>
+        {
+            options.TimestampFormat = "[yyyy/MM/dd HH:mm:ss] ";
```
Comment (externl) [inline src/Hello/Hello.cs:39]: The extra space on the end is required and not a typo.

Comment (bernardnormier) [reply src/Hello/Hello.cs:39]: What about including milliseconds in this timestamp?

https://learn.microsoft.com/en-us/dotnet/standard/base-types/how-to-display-milliseconds-in-date-and-time-values

File: src/Hello/Hello.cs [hunk 2]
```diff
@@ -34,7 +34,10 @@
 // Create a simple console logger factory and configure the log level for category IceRpc.
 using ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
     builder
-    .AddSimpleConsole()
+    .AddSimpleConsole(options =>
+        {
+            options.TimestampFormat = "[yyyy/MM/dd HH:mm:ss.fff] ";
+        })
```
Comment (pepone) [inline src/Hello/Hello.cs:40]:
```suggestion
    .AddSimpleConsole(options => options.TimestampFormat = "[yyyy/MM/dd HH:mm:ss.fff] ")
```

**Review by pepone — APPROVED** [review]
Looks good. Just a minor suggestion for the syntax.

**Pull request merged**